tests/integration/api/test_ruleset.py

```diff
@@ -0,0 +1,129 @@
+import asyncio
+from unittest.mock import patch
+
+import aiodocker
+
+from eda_server import ruleset
+from eda_server.config.enums import DeploymentTypeEnum
+
+activation_id = 1
+large_data_id = 1
+execution_environment = "fedora:36"
+rulesets = ""
+ruleset_sources = []
+inventory = ""
+extravars = ""
+working_directory = "/tmp"
+host = "test-eda-server"
+port = 15000
+
+
+class TstDocker(aiodocker.docker.Docker):
+    def __init__(self, *args, **kwargs):
```

Comment:
If this is a mock class, why does it inherit real `aiodocker.docker.Docker` class?

Reply:
Was probably going to check something with `isinstance` that didn't become realized.